### PR TITLE
chore: upgrade dependencies

### PR DESCRIPTION
**Pull request by sonle-geekyants**
**- What I did**
- Upgraded dependencies to the latest

**- How I did it**

**- How to verify it**

**- Description for the changelog**

### CODE REVIEW

File: lib/screens/edit_persona/edit_category_fields.dart [hunk 1]
```diff
@@ -415,9 +415,6 @@ class _EditCategoryFieldsState extends State<EditCategoryFields> {
   Widget inputField(BasicData basicData, {bool isCustomField = false}) {
     return Slidable(
       key: UniqueKey(),
-      actionPane: SlidableDrawerActionPane(),
```
Comment (sachins-geekyants) [inline lib/screens/edit_persona/edit_category_fields.dart:418]: Do we not need `SlidableDrawerActionPane ` anymore ?

Comment (sachins-geekyants) [reply lib/screens/edit_persona/edit_category_fields.dart:418]: Is there any alternate UI which has to be rendered ?

File: pubspec.yaml [hunk 2]
```diff
@@ -30,73 +30,68 @@ dependencies:
...
-  #   git:
-  #     url: https://github.com/atsign-foundation/at_widgets.git
-  #     path: packages/at_onboarding_flutter
-  #     ref: trunk
+  http: ^1.2.1
+  package_info_plus: ^5.0.0
+  web: ^0.5.0
+  file_picker: ^6.1.1
+  flutter_inappwebview: ^6.0.0
+  at_onboarding_flutter:
```
Comment (sachins-geekyants) [inline pubspec.yaml]: Can it be replaced with hosted version ?

File: lib/screens/edit_persona/edit_category_fields.dart (hunk 1 above)
Comment (sonle-geekyants) [reply lib/screens/edit_persona/edit_category_fields.dart:418]: @sachins-geekyants The package has changed its usage and there will be no alternate UI

File: pubspec.yaml (hunk 2 above)
Comment (sonle-geekyants) [reply pubspec.yaml]: @sachins-geekyants It can be if you want

Comment (sachins-geekyants) [reply pubspec.yaml]: Let's use hosted version unless there is something from trunk branch that we need

Comment (sonle-geekyants) [reply pubspec.yaml]: @sachins-geekyants Done

**Review by sachins-geekyants — APPROVED** [review]

**Pull request merged**